account/views.py

```diff
@@ -215,6 +215,37 @@ def get(self, request):
             pass
 
         return Response({"message": "No Information on Account"}, status.HTTP_200_OK)
+    
+    def put(self, request):
+        user = request.user
+        if user.is_admin_user:
+            serializer = AdminSerializer(user, data=request.data, partial=True)
+            if serializer.is_valid(raise_exception=True):
+                serializer.save()
+                return Response(serializer.data, status=status.HTTP_200_OK)
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        
+        try:
+            lecturer = Lecturer.objects.get(user=user)
+            serializer = LecturerSerializer(lecturer, data=request.data, partial=True)
+            if serializer.is_valid(raise_exception=True):
+                serializer.save()
+                return Response(serializer.data, status=status.HTTP_200_OK)
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        except Lecturer.DoesNotExist:
+            pass
```

Comment:
Handle this exception properly.

Reply:
Thats the best way I can handle the exception, so the other part of the code can be reacheable.